lighthouse-cli/run.ts

```diff
@@ -27,13 +29,36 @@ interface LighthouseError extends Error {
   code?: string
 }
 
+function parseChromeFlags(flags: string) {
+  let args = yargs(flags).argv;
+  const allKeys = Object.keys(args);
+
+  // Remove any "mangled" arguments (args that contain dashes are duplicated as
+  // camelCased variants, but the dashed version remains)
+  let keysToFilter = allKeys.filter(k => k.indexOf('-') !== -1)
```

Comment:
could also export this function separately and throw a test on here if you want to make sure it keeps the same functionality